ui-admin/src/api/api.tsx

```diff
@@ -913,6 +917,12 @@ export default {
     return await this.processJsonResponse(response)
   },
 
+  async fetchInternalConfig(): Promise<InternalConfig> {
+    const url = `${API_ROOT}/internal/v1/config`
```

Comment:
How will this work when we add additional, non-kit config to the UI?
Does it make sense to make this kit specific now?

Reply:
the idea is there is a single config endpoint -- I don't want the API endpoint to dictate how we slice the UI, and some config values may be applicable to multiple integrations.  So this endpoint will be unchanged, we'll just add ux pages (and update the configMap on the backend with the values we want to expose)